onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1037,9 +1037,8 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                "T",
                OpSchema::Optional)
         .Input(5,
-               "past_sequence_length",
-               "When buffered past_key and past_value is used (present_key uses same tensor as past_key), required"
-               "to specify past_sequence_length (could be 0). Otherwise, past_sequence_length inferred from past_key.",
+               "attention_mask",
+               "2d Tensor of shape (batch_size, past_sequence_length + sequence_length). Must be a right padding mask.",
```

Comment:
Could we support both left side and right side padding?

Reply:
Flash Attention does not support left padding unfortunately

Reply:
It's fine to throw Not Implemented exception in CUDA. For the interface, we can describe that it supports left padding (which is needed by DML).

Reply:
we can do this later, but not for this bug fix.